examples/diagnostic_settings/_header.md

```diff
@@ -0,0 +1,4 @@
+# Default example
```

Comment:
I'd change the name of this since it is the `Diagnostic Settings Example`, and there is already a default example - probably need to regenerate the docs as well.

Reply:
also - you may want to add the note that diagnostic settings are not available in all regions in here as well - I know it is in the main.tf in a comment but adding it here may make it more visible.